src/schema/me/create_credit_card_mutation.js

```diff
@@ -0,0 +1,57 @@
+import { GraphQLInt, GraphQLObjectType, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { GravityIDFields } from "schema/object_identification"
+
+export const CreditCardType = new GraphQLObjectType({
+  name: "CreditCard",
+  fields: () => ({
+    ...GravityIDFields,
+    brand: {
+      type: GraphQLString,
+      description: "Brand of credit card",
+    },
+    name: {
+      type: GraphQLString,
+      description: "Name on the credit card",
+    },
+    last_digits: {
+      type: GraphQLString,
+      description: "Last four digits on the credit card",
+    },
+    expiration_month: {
+      type: GraphQLInt,
+      description: "Credit card's expiration month",
+    },
+    expiration_year: {
+      type: GraphQLInt,
+      description: "Credit card's expiration year",
+    },
+  }),
+})
+
+export default mutationWithClientMutationId({
+  name: "CreditCard",
+  decription: "Create a credit card",
+  inputFields: {
+    token: {
+      type: GraphQLString,
+    },
+  },
+  outputFields: {
+    credit_card: {
+      type: CreditCardType,
+      resolve: credit_card => credit_card,
```

Comment:
Nice, yea, unless you _needed_ to refetch might as well use the mutation response directly 👍